default-experiment-descriptor.json

```diff
@@ -3,6 +3,11 @@
   "label": "",
   "basedir": ".",
   "LaTeXinMD": false,
+  "code-assessment": {

```

Comment:
default experiment descriptor should have the code editor as false. The developer will turn it to true if they want

Reply:
done